lib/rust/parser/debug/tests/parse.rs

```diff
@@ -865,6 +865,23 @@ fn method_app_in_minus_unary() {
         (UnaryOprApp "-" (OprApp (Ident Number) (Ok ".") (Ident positive_infinity))));
 }
 
+#[test]
+fn autoscope_operator() {
+    test!("x : ..True", (TypeSignature (Ident x) ":" (AutoscopedIdentifier ".." True)));
+    test!("x = ..True", (Assignment (Ident x) "=" (AutoscopedIdentifier ".." True)));
+    test!("x = f ..True",
+        (Assignment (Ident x) "=" (App (Ident f) (AutoscopedIdentifier ".." True))));
+    expect_invalid_node("x = ..4");
+    expect_invalid_node("x = ..Foo.Bar");
```

Comment:
Reading the rest of the code I assume `x = ..(a + b)` is also going to pass `expect_invalid_node` test.

Reply:
Yeah, the type cannot be constructed without an identifier token, so as long as the test covers the handling of any non-identifier operand, it effectively covers all non-identifier operands.